src/Model/Contact/Relation.php

```diff
@@ -272,20 +281,30 @@ static public function getSuggestions(int $uid, int $start = 0, int $limit = 80)
 
 		// The query returns contacts where contacts interacted with whom the given user follows.
 		// Contacts who already are in the user's contact table are ignored.
-		$results = DBA::select('contact', [],
-			["`id` IN (SELECT `cid` FROM `contact-relation` WHERE `relation-cid` IN
+		$results = DBA::select('contact', [], ["`id` IN (SELECT `cid` FROM `contact-relation` WHERE `relation-cid` IN
 				(SELECT `cid` FROM `contact-relation` WHERE `relation-cid` = ?)
 					AND NOT `cid` IN (SELECT `id` FROM `contact` WHERE `uid` = ? AND `nurl` IN
 						(SELECT `nurl` FROM `contact` WHERE `uid` = ? AND `rel` IN (?, ?))))
 			AND NOT `hidden` AND `network` IN (?, ?, ?, ?)",
-			$cid, 0, $uid, Contact::FRIEND, Contact::SHARING,
-			Protocol::ACTIVITYPUB, Protocol::DFRN, $diaspora, $ostatus],
-			['order' => ['last-item' => true], 'limit' => $totallimit]
+			$cid,
+			0,
+			$uid,
+			Contact::FRIEND,
+			Contact::SHARING,
+			Protocol::ACTIVITYPUB,
+			Protocol::DFRN,
+			$diaspora,
+			$ostatus,
```

Comment:
Please put the parameter values on the same line when their `?` placeholders also are on the same line in the query. This helps with visually mapping values to placeholders.
```suggestion
			$cid,
			0,
			$uid, Contact::FRIEND, Contact::SHARING,
			Protocol::ACTIVITYPUB, Protocol::DFRN, $diaspora, $ostatus,
```